packages/protobuf/src/index.ts

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+export type { ProtoRuntime } from "./private/proto-runtime.js";
```

Comment:
Not sure if you want to export like that from a "private" module.

Reply:
Would be best to split into a ./proto-runtime.ts with just the interface. But with `typeof protoN` (see typescript.ts), we don't need this!